packages/sanity/src/structure/panes/document/DocumentPaneProvider.tsx

```diff
@@ -748,12 +748,18 @@ export const DocumentPaneProvider = memo((props: DocumentPaneProviderProps) => {
   )
 
   useEffect(() => {
+    let timeout: NodeJS.Timeout
```

Comment:
```suggestion
    let timeout: ReturnType<typeof setTimeout>
```
removes a dependency on NodeJS types in React components

Reply:
Done :)